tools/upload_text.php

```diff
@@ -167,80 +148,66 @@
     exit;
 }
 
+$return_anchor = "<a href='$back_url'>$back_blurb</a>";
+$return_message = "<p>". sprintf(_("Return to the %s"), $return_anchor). "</p>";
+
 if (!isset($action))
 {
     // Present the upload page.
-
-    output_header($title);
```

Comment:
We should keep the themed output on this page.

Reply:
Done this but I haven't put the statsbar back.